src/vivarium_public_health/disease/special_disease.py

```diff
@@ -86,25 +91,30 @@ class RiskAttributableDisease(Component):
         recoverable : True
     """
 
-    CONFIGURATION_DEFAULTS = {
-        "risk_attributable_disease": {
-            "threshold": None,
-            "mortality": True,
-            "recoverable": True,
-        }
-    }
-
     ##############
     # Properties #
     ##############
 
     @property
     def name(self):
-        return f"disease_model.{self.cause.name}"
+        # return f"disease_model.{self.cause.name}"
+        return self.cause.name
```

Comment:
We probably want some reference to "risk_attributable_disease" in this name. Otherwise we may easily run into namespace collisions.